packages/dropzone/src/solid.tsx

```diff
@@ -192,7 +186,7 @@ export function createDropzone(_props: DropzoneOptions) {
       }
     });
 
-    if (isValidQuantity(acceptedFiles, props.multiple, props.maxFiles)) {
+    if (!isValidQuantity(acceptedFiles, props.multiple, props.maxFiles)) {
```

Comment:
whoopsies should prob be fixed in a PR that'll get merged in the near future cc @markflorkowski